plugins/generator-1.19.2/forge-1.19.2/utils/mcelements.ftl

```diff
@@ -34,4 +34,12 @@
 
 <#function toBlockPos x y z>
     <#return "new BlockPos(" + opt.removeParentheses(x) + "," + opt.removeParentheses(y) + "," + opt.removeParentheses(z) +")">
+</#function>
+
+<#function toOptionalVec3i x y z trailingComma=false>
```

Comment:
Are there any other potential locations where this would be used, otherwise it may be better to inline this

Reply:
Not that I can think of